startupscript/butane/wb/configure.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# configure.sh - Workbench CLI Setup and Configuration
+#
+# This script performs initial setup and configuration for the Workbench CLI,
+# preparing it for future runs via wb.sh. It handles:
+# 1. Building the workbench Docker container with server configuration
+# 2. Authenticating with the Workbench server
+# 3. Setting up the default workspace context
+#
+# The script uses metadata values to configure:
+# - terra-cli-server: The Workbench server endpoint
+# - terra-workspace-id: The default workspace to use
+#
+# This is typically run once during environment setup to preconfigure
+# the workbench for subsequent interactive use.
+
```

Comment:
can this be part of the 00x- script so it's clear when it's executed in the app start up systemd.

Reply:
ah sad will need to update [this ](https://github.com/verily-src/verily1/blob/90f0e11b0de2880ee83683720d3abb155692a446/workbench/app-manager/static/butane/butane.yaml.tmpl#L97). just fyi we bump the version automatically with a pr once a day and i'm ok dev is broken for a short bit.